Lombiq.OrchardCoreApiClient.Tests.UI/Recipes/Lombiq.OrchardCoreApiClient.Tests.UI.OpenId.recipe.json

```diff
@@ -21,24 +21,30 @@
     },
     {
       "name": "OpenIdServerSettings",
-      "TestingModeEnabled": false,
-      "AccessTokenFormat": "JsonWebToken",
-      "Authority": "",
-      "SigningCertificateStoreLocation": "",
-      "SigningCertificateStoreName": "",
-      "SigningCertificateThumbprint": "",
-      "EncryptionCertificateStoreLocation": "",
-      "EncryptionCertificateStoreName": "",
-      "EncryptionCertificateThumbprint": "",
+      "AccessTokenFormat": 0,
+      "Authority": null,
+      "DisableAccessTokenEncryption": false,
+      "EncryptionCertificateStoreLocation": null,
+      "EncryptionCertificateStoreName": null,
+      "EncryptionCertificateThumbprint": null,
+      "SigningCertificateStoreLocation": null,
+      "SigningCertificateStoreName": null,
+      "SigningCertificateThumbprint": null,
       "EnableTokenEndpoint": true,
       "EnableAuthorizationEndpoint": false,
       "EnableLogoutEndpoint": false,
       "EnableUserInfoEndpoint": false,
+      "EnableIntrospectionEndpoint": false,
+      "EnableRevocationEndpoint": false,
       "AllowPasswordFlow": false,
       "AllowClientCredentialsFlow": true,
       "AllowAuthorizationCodeFlow": false,
       "AllowRefreshTokenFlow": false,
-      "AllowImplicitFlow": false
+      "AllowHybridFlow": false,
+      "AllowImplicitFlow": false,
+      "DisableRollingRefreshTokens": true,
+      "RequireProofKeyForCodeExchange": false,
+      "UseReferenceAccessTokens": true
     },
```

Comment:
I've changed the configuration in the admin menu, exported it into a recipe and pasted it here. The original recipe actually had some properties that `OpenIdServerSettings` doesn't have (any more?) which is why there are so many differences.